app/jobs/webhooks/instagram_events_job.rb

```diff
@@ -18,18 +14,46 @@ def perform(entries)
     end
   end
 
-  # @see https://developers.facebook.com/docs/messenger-platform/instagram/features/webhook
+  # https://developers.facebook.com/docs/messenger-platform/instagram/features/webhook
   def process_entries(entries)
     entries.each do |entry|
-      entry = entry.with_indifferent_access
-      messages(entry).each do |messaging|
-        send(@event_name, messaging) if event_name(messaging)
-      end
+      process_single_entry(entry.with_indifferent_access)
     end
   end
 
   private
 
+  def process_single_entry(entry)
+    process_messages(entry)
+  end
+
+  def process_messages(entry)
+    messages(entry).each do |messaging|
+      Rails.logger.info("Instagram Events Job Messaging: #{messaging}")
+
+      instagram_id, _contact_id = instagram_and_contact_ids(messaging)
```

Comment:
instagram channel id ? not seeming to use the contact id

Reply:
Done